.gitignore

```diff
@@ -1,4 +1,6 @@
 aws-stepfunctions-local-credentials.txt
 .venv
+.metals
```

Comment:
Great catch! That's an entirely-unneeded Scala server extension that I forgot to disable for our workspace in VSCode (although why I have to disable it in a Python project is a head-scratcher). Updated this to remove the ignore since it's no longer needed by myself 😆 
https://github.com/department-of-veterans-affairs/Virtual-Regional-Office/pull/15/commits/ff53847da34e1b814befadef604037dd24045b9a#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L3

Reply:
(Resolved)